docker/maistra-builder_1.1.Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN chmod -R +x /tmp/scripts/ && \
     /tmp/scripts/install_shellcheck.sh && \
     /tmp/scripts/install_kubectl.sh && \
     /tmp/scripts/install_docker.sh && \
+    /tmp/scripts/install_docs_tools.sh \
```

Comment:
this seems to be missing a `&&` at the end - this way we're just passing the `rm` command as an arg to the script

Reply:
Good catch :) Built properly but I hadn't checked to see if the directory exists.

Reply:
Fixed